app/code/Magento/CustomerGraphQl/etc/schema.graphqls

```diff
@@ -16,15 +16,15 @@ type Mutation {
 }
 
 input CustomerAddressInput {
```

Comment:
This input type is used for `createCustomerAddress` as well as `updateCustomerAddress` where all these fields are not required.